.scalafmt.conf

```diff
@@ -1,23 +1,45 @@
-version = 2.7.5
+version = 3.0.8
 
 style = default
 
 maxColumn = 100
+docstrings.wrap = false
 
 // Vertical alignment is pretty, but leads to bigger diffs
 align.preset = none
 
+# Commented-out configs are Good Ideas but will create a big initial diff if applied
+
 danglingParentheses.preset = false
+# danglingParentheses.preset = true
 
 rewrite.rules = [
   AvoidInfix
   RedundantBraces
   RedundantParens
-  AsciiSortImports
   PreferCurlyFors
+  # SortModifiers
+]
+
+rewrite.sortModifiers.order = [
+  override, implicit, private, protected, final, sealed, abstract, lazy
 ]
 
+# rewrite.trailingCommas.style = multiple
```

Comment:
This reduces diffs in the long run. :+1:

Reply:
`always` would reduce them even more but is terribly, terribly ugly.